test/functional/python/test_spglib.py

```diff
@@ -2,7 +2,7 @@
 import unittest
 
 import numpy as np
-import yaml
```

Comment:
`pip install types-PyYAML` did not work somehow.

Reply:
It has to be in `pre-commit-config.yaml`

Reply:
Thanks!